index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
     </section>
   </section>
 
+  <section class="appendix">
+    <h2>Proof Format Trade-Offs</h2>
+
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging syntaxes and digital proof formats. Each
+format has benefits and drawbacks. The following table is intended to summarize
+a number of these native trade-offs.
+    </p>
+
+    <p>
+The table below compares three proof format ecosystems; JSON+JWTs,
+JSON-LD+JWTs, and JSON-LD+LD-Proofs. At the time of publication of this
+specification, no organization is planning to use JSON+JWTs. The existence
+of JSON+JWTs in the table below is provided based on review comments on the
+specification for readers that were curious about the benefits that JSON-LD
+provided over JSON. Readers should also be aware that Zero-Knowledge Proofs
+are currently proposed as a sub-type of LD-Proofs and thus fall into the
+final colum below.
+    </p>
+
+    <table class="simple">
+      <thead>
+        <tr>
+          <th style="text-align: center;">Feature</th>
+          <th style="text-align: center;">JSON<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>JWTs</th>
+          <th style="text-align: center;">JSON&#8209;LD<br>+<br>LD&#8209;Proofs</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+<a href="#pf1">PF1.</a> Support for open world data modelling.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf2">PF2.</a> Universal identifier mechanism for JSON objects via
+the use of URIs.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf3">PF3.</a> A way to disambiguate keys shared among different
+JSON documents by mapping them to IRIs via a context.
+          </td>
+          <td class="missing">✖</td>
```

Comment:
When do you need this within a VC? This is a feature of DID documents I believe.

Reply:
Ah, I don't mean "keys" as in the "cryptographic key" sense. I mean "keys" as in Javascript Object properties -- key-value pairs -- sense. You are right, the language is confusing. The JSON spec calls these "names", which is also confusing. I've tried to apply a fix by using "properties", as in "property-value" statements in 43ff3769938b324059b9f9681c878a2c81d78a7d.

Reply:
Also, I hope those reading this particular comment thread appreciate the irony that both @msporny and @pelle were miscommunicating specifically because English doesn't provide a `@context` feature either.